appuio/signalilo/templates/serviceaccount.yaml

```
@@ -0,0 +1,10 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: signalilo
```

Shouldn't this be

```suggestion
  name: {{ include "signalilo.serviceAccountName" }}
```

since the documentation states that the chart will use the name generated from the full name template, if `serviceAccount.name` isn't set and `serviceAccount.create` is true.

---

Thank you for reviewing and you are absolutely right. F... copy&paste. Corrrected it.